Source/JavaScriptCore/llint/InPlaceInterpreter.asm

```diff
@@ -5549,6 +5633,24 @@ elsif X86_64
 end
 end
 
+_ipint_call_impl:
+    # 0 - 3: function index
```

Comment:
Side comment: we really need a better way to specify these offsets

Reply:
Yeah, IPInt has the old LLInt problem that offsets are hardcoded instead of derived from LLIntOffsetsExtractor